src/main/java/com/joy/joyfillandroid/ShortField.kt

```diff
@@ -0,0 +1,77 @@
+package com.joy.joyfillandroid
+
+import android.annotation.SuppressLint
+import android.content.Context
+import android.graphics.Canvas
+import android.graphics.Color
+import android.text.InputType
+import android.view.Gravity
+import android.widget.EditText
+import android.widget.LinearLayout
+import androidx.core.content.ContextCompat
+
+@SuppressLint("AppCompatCustomView", "ResourceAsColor")
+class ShortField(context: Context) : EditText(context) {
+
+
+    init {
+        val textField = TextField(context)
+        val linearLayout = LinearLayout(getContext())
```

Comment:
Can you explain the difference of `getContext` vs the line above `context` getting passed into this class?

If they do different things we would want to name them.